src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -643,6 +643,42 @@ const convertChunkToContentBlocks = (chunk: Chunk): ?Array<BlockNodeRecord> => {
     const {depth, type, parent} = block;
 
     const key = block.key || generateRandomKey();
+    let parentTextNodeKey = null; // will be used to store container text nodes
+
+    // childrens add themselves to their parents since we are iterating in order
+    if (parent) {
+      const parentIndex = acc.cacheRef[parent];
+      let parentRecord = acc.contentBlocks[parentIndex];
+
+      // if parent has text we need to split it into a separate unstyled element
+      if (parentRecord.getChildKeys().isEmpty() && parentRecord.getText()) {
```

Comment:
If I'm reading this right, then this conditional will be true whenever the parent has no children and has any text. It will take all the text and put it into a textNode, and add that to the children.

It seems this will happen even for `<blockquote>hi</blockquote>` as well as `<blockquote>hi <h1>world</h1></blockquote>`. Why do it this way? Maybe more consistent?